debug-10-nodes.patch

```diff
@@ -1,10 +1,13 @@
 diff --git a/src/config/index.ts b/src/config/index.ts
-index fda0dd1..0d19803 100644
+index ba78321f..7fe17d9d 100644
```

Comment:
Probably should make a new patch & PR this seperately. (or instead of using patch, make a new config file that can be turned on via env var or something like that).